state-sync/state-sync-v2/state-sync-driver/src/notification_handlers.rs

```diff
@@ -393,12 +391,17 @@ impl FusedStream for ErrorNotificationListener {
 #[derive(Clone)]
 pub struct MempoolNotificationHandler<M> {
     mempool_notification_sender: M,
+    state_sync_driver_config: StateSyncDriverConfig,
 }
 
 impl<M: MempoolNotificationSender> MempoolNotificationHandler<M> {
-    pub fn new(mempool_notification_sender: M) -> Self {
+    pub fn new(
+        mempool_notification_sender: M,
+        state_sync_driver_config: StateSyncDriverConfig,
```

Comment:
nit: FWIW, I wouldn't pass the entire config in here because we're only really using one item. I'd rather take the value directly 😄 (e.g., `mempool_commit_ack_timeout_ms`).

But, I guess some could argue this a style choice. So, I'll leave it up to you 😄

Reply:
Done.